.markdownlint.yaml

```diff
@@ -17,3 +17,6 @@ MD026:
 
 # MD028/no-blanks-blockquote
 MD028: false
+
+# MD060/table-column-style
+MD060: false
```

Comment:
This is a good rule to have. We should fix the broken tables instead, or change the rule to a different style that fits our markdown. I'm against disabling it entirely.